apps/src/applab/designElements/library.js

```diff
@@ -170,9 +175,12 @@ export default {
   },
 
   getCurrentTheme: function(parentScreen) {
+    const defaultThemeIndex = experiments.isEnabled('applabThemes')
+      ? DEFAULT_THEME_INDEX
+      : CLASSIC_THEME_INDEX;
     return parentScreen
       ? parentScreen.getAttribute('data-theme')
-      : themeOptions[DEFAULT_THEME_INDEX];
+      : themeOptions[defaultThemeIndex];
   },
 
   applyCurrentTheme: function(element, parentScreen) {
```

Comment:
I'd prefer to leave it since we call it with `designMode.activeScreen()` in 6 places and `Applab.activeScreen()` in 4 places. In other words, it is important to get the right value based on how it is being used.
The latter is used when creating elements with the runtime API (in `commands.js`). I'm not a fan of using the global, but it's the only function currently exposed that tells you the active screen at runtime.

Reply:
When does `designMode.activeScreen()` not equal `Applab.activeScreen()`?